content/tutorials/integration-tutorials/keras_models.md

```diff
@@ -7,22 +7,19 @@ title: Keras models
 ---
 
 {{< cta-button colabLink="https://colab.research.google.com/github/wandb/examples/blob/master/colabs/keras/Use_WandbModelCheckpoint_in_your_Keras_workflow.ipynb" >}}
-Use Weights & Biases for machine learning experiment tracking, dataset versioning, and project collaboration.
 
-{{< img src="/images/tutorials/huggingface-why.png" alt="" >}}
-
-This Colab notebook introduces the `WandbModelCheckpoint` callback. Use this callback to log your model checkpoints to Weight and Biases [Artifacts]({{< relref "/guides/core/artifacts/" >}}).
+This notebook introduces the `WandbModelCheckpoint` callback. Use this callback
+to log your model checkpoints to Weight and Biases
+[Artifacts]({{< relref "/guides/core/artifacts/" >}}).
 
 ## Setup and Installation
 
-First, let us install the latest version of Weights and Biases. We will then authenticate this colab instance to use W&B.
-
+First, let us install the latest version of Weights and Biases and authenticate.
```

Comment:
The same feedback as in the Hugging Face example for similar changes here and in the following pages.